src/test/java/bio/terra/service/filedata/azure/util/BlobIOTestUtility.java

```diff
@@ -25,19 +27,19 @@
  * copy operations.
  */
 public class BlobIOTestUtility {
+  public static final long MiB = 1024 * 1024;
```

Comment:
Constants should be uppercase. Also there is a spring builtin that can be used to generate megabyte values: `DataSize.ofMegabytes(1).toBytes();`

Also, looking at usage, it is always used as `MiB / 10`, so that should be the constant instead, if a constant is needed at all.

Reply:
I want to keep it as constant (particular if we add a scale test). It's easier to have the constant now rather than go back and change things to use use the constant.

Re: the Spring util, given that this will eventually move to CRL (or some equivalent) I'd like to avoid any dependencies on Spring